packages/safe-apps-sdk/package.json

```diff
@@ -18,14 +20,24 @@
   ],
   "scripts": {
     "test": "jest",
-    "format-dist": "sed -i '' 's/\"files\":/\"_files\":/' dist/package.json",
-    "build": "yarn rimraf dist && tsc && yarn format-dist"
+    "prebuild": "node -p \"'export const getSDKVersion = () => \\'' + require('./package.json').version.split('-')[0] + '\\';'\" > src/version.ts",
+    "build": "yarn rimraf ./dist && yarn build:cjs && yarn build:esm && yarn build:types",
+    "build:cjs": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs && echo '{\"type\": \"commonjs\"}' > ./dist/cjs/package.json",
+    "build:esm": "tsc --project tsconfig.json --module es2015 --outDir ./dist/esm && echo '{\"type\": \"module\"}' > ./dist/esm/package.json",
```

Comment:
Currently we are aligning all packages and apps to use at least ES2021.

Is supported by most used browsers since 2020 and by Node 16, while we also aim to move everything to Node 18 soon

Reply:
Ok, I put ES2020 since ES2021 isn't a valid value TS supports.